base/distributed/cluster.jl

```diff
@@ -167,8 +175,6 @@ function start_worker(out::IO, cookie::AbstractString)
     print(out, LPROC.bind_addr)
     print(out, '\n')
     flush(out)
-    # close STDIN; workers will not use it
```

Comment:
Thanks for digging that out. @vtjnash can you think of any reasons now to not close STDIN on the workers?